src/prop/cadical.cpp

```diff
@@ -204,6 +204,26 @@ unsigned CadicalSolver::getAssertionLevel() const
 
 bool CadicalSolver::ok() const { return d_inSatMode; }
 
+void CadicalSolver::setDrat()
+{
+  sprintf(d_tempDratFilePath, "dratProof_%d.drat", getpid());
```

Comment:
This is only safe if the string length does not exceed 256. Better use `std::string` and `std::stringstream` to construct the string.